### PR TITLE
Updated Tooltips based on Sarah's input

### PR DESCRIPTION
**Pull request by bencloward**
### Purpose of this PR
Updated node tooltips based on Sarah W's feedback

### CODE REVIEW

**Review by tracychen1234 — APPROVED** [review]
Looks good!

File: com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Input/Basic/ConstantNode.cs [hunk 1]
```diff
@@ -64,7 +64,7 @@ internal class ConstantNode : IStandardNode
         public static NodeUIDescriptor NodeUIDescriptor => new(
             Version,
             Name,
-            tooltip: "constants pi, tau, phi, e, or sqrt2",
+            tooltip: "Returns the selected constant value (pi, tau, phi, e, or sqrt2)",
```
Comment (sarah-welton) [inline com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Input/Basic/ConstantNode.cs:67]:
```suggestion
            tooltip: "Returns the selected constant value (pi, tau, phi, e, or sqrt2).",
```

File: com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/Logic/AllNode.cs [hunk 2]
```diff
@@ -19,7 +19,7 @@ internal class AllNode : IStandardNode
         public static NodeUIDescriptor NodeUIDescriptor => new(
             Version,
             Name,
-            tooltip: "returns true if all components of the input In are non-zero",
+            tooltip: "Checks if all components of the input *In* are non-zero values",
```
Comment (sarah-welton) [inline com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/Logic/AllNode.cs:22]:
```suggestion
            tooltip: "Checks if all components of the input In are non-zero values",
```

We didn't keep this up elsewhere, so unless we're bolding all port names, let's just leave this for now.

**Review by sarah-welton — CHANGES_REQUESTED** [review]
Couple minor things, just used suggest changes so they should be easy to resolve :) Thanks!

**Review by sarah-welton — APPROVED** [review]
LGTM! :)

**Pull request merged**